packages/app/src/app/components/Preview/index.js

```diff
@@ -50,6 +50,7 @@ type State = {
   overlayMessage: ?string,
   hibernated: boolean,
   sseError: boolean,
+  localClose: boolean,
```

Comment:
What does `localClose` mean? Maybe we could add a comment describing what it does.